### ISSUE TITLE
Support skip the function "checkValueNotNull" of check value not null by job configuration

### ISSUE THREAD

**Issue by haosky** [pingcap/tispark#2087]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by shiyuhang0** [comment]
Good idea, but I am bust recently, I will work for it once I have time.
